modules/mongodb/manifests/python.pp

```diff
@@ -1,4 +1,7 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
+# == Class: mongodb::python
+#
+# Installs PyMongo package
```

Comment:
This should be "Manages the python mongo client" so we can change the underlying pip package as needed.